### ISSUE TITLE
Loadbalancer service is not working in kubernetes 1.6

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#8532]
**Rancher Versions:**
Server: master build
kubernetes (if applicable): 1.6

**Steps to Reproduce:**
- Install kubernetes 1.6 with rancher as cloud provider
- Run the following yaml file:
```
apiVersion: v1
kind: ReplicationController
metadata:
  name: nginx
spec:
  replicas: 1
  selector:
    name: nginx
  template:
    metadata:
      labels:
        name: nginx
    spec:
      containers:
        - name: nginx
          image: husseingalal/nginx-curl
          ports:
            - containerPort: 80

---

apiVersion: v1
kind: Service
metadata:
  name: lbnginx
  labels:
    name: lbnginx
spec:
  type: LoadBalancer
  ports:
  - port: 8888
    targetPort: 80
  selector:
    name: nginx
```

**Results:** 

The load balancer container isn't created: 
```
{
    "apiVersion": "v1",
    "kind": "Service",
    "metadata": {
        "creationTimestamp": "2017-04-18T00:40:33Z",
        "labels": {
            "name": "lbnginx"
        },
        "name": "lbnginx",
        "namespace": "default",
        "resourceVersion": "11053",
        "selfLink": "/api/v1/namespaces/default/services/lbnginx",
        "uid": "a1cb9f57-23cf-11e7-9be1-025df495afe9"
    },
    "spec": {
        "clusterIP": "10.43.250.135",
        "ports": [
            {
                "nodePort": 31066,
                "port": 8888,
                "protocol": "TCP",
                "targetPort": 80
            }
        ],
        "selector": {
            "name": "nginx"
        },
        "sessionAffinity": "None",
        "type": "LoadBalancer"
    },
    "status": {
        "loadBalancer": {}
    }
}
```

**Comment by deniseschannon** [comment]
Fixed in `rancher/k8s:v1.6.2-rancher3-1`

**Comment by galal-hussein** [comment]
Rancher Version: v1.6.0-rc4
Kubernetes image: rancher/k8s:v1.6.2-rancher3-1

I was able to create and test Load Balancer service and i can see an endpoint ip being assigned successfully.
